modules/tutorials/pages/java-tutorial/install-couchbase-java-sdk.adoc

```diff
@@ -0,0 +1,76 @@
+:page-pagination:
+
+= Installing the Couchbase Java SDK
+
+:description: pass:q[In this tutorial, you're going to create a skeleton application for interacting with the student database you created in xref:install-couchbase-server.adoc[]]
+
+[abstract]
+{description}
+
+== Prerequisites
+
+Java is a fairly popular programming language these days, so we're going to use it to build our student/course application. To keep things as light as possible (this is a tutorial, after all), we're not going to worry about building a web front end or REST service, just a few methods to read/write our documents to the database.
+
+You will need a few things installed on your machine before you begin:
+
+* The Java Software Development Kit (version {minimum-jdk-version})
```

Comment:
these 2 versions haven't rendered on the sample output

Reply:
Good spot!